automated install/basic-install.sh

```diff
@@ -24,7 +24,7 @@ setupVars=/etc/pihole/setupVars.conf
 
 webInterfaceGitUrl="https://github.com/pi-hole/AdminLTE.git"
 webInterfaceDir="/var/www/html/admin"
-piholeGitUrl="https://github.com/pi-hole/pi-hole.git"
+piholeGitUrl="https://github.com/tuplink/pi-hole.git"
```

Comment:
You already did, those were outdated commits, the latest version you have @ [2d7e316] takes care of that part.